cpp_exercises/day2/date/date.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
-#include<iosfwd>
+#include <QtGlobal>
```

Comment:
Visto che siamo in C++ standard (ie. senza Qt), userei `<stdint.h>` e `int64_t` come tipi al posto di `qint64`